indico/modules/events/registration/controllers/management/__init__.py

```diff
@@ -27,6 +28,9 @@ class RHManageRegFormsBase(RHConferenceModifBase):
 
     CSRF_ENABLED = True
 
+    def _process(self):
+        return RH._process(self)
```

Comment:
yay for terrible legacy stuff (if we have time in the end we should check if we can remove the stupid `_process` from `RHConferenceModifBase`)